crates/nu-cmd-extra/src/extra/bits/xor.rs

```diff
@@ -42,13 +59,37 @@ impl Command for BitsXor {
         input: PipelineData,
     ) -> Result<PipelineData, ShellError> {
         let head = call.head;
-        let target: i64 = call.req(engine_state, stack, 0)?;
-        // This doesn't match explicit nulls
+        let target: Value = call.req(engine_state, stack, 0)?;
+        let endian = call.get_flag::<Value>(engine_state, stack, "endian")?;
```

Comment:
The same here: `call.get_flag::<Spanned<String>>(engine_state, stack, "endian")`